config.json

```diff
@@ -256,6 +256,22 @@
         "math"
       ]
     },
+    {
+      "slug": "kindergarten-garden",
+      "uuid": "0b92ffee-c092-4ab1-ad78-76c8cf80e1b5",
```

Comment:
Hi @lathspell - just to make sure, this is a brand new UUID and hasn't been taken from the Java track?